ui-specs/README.md

```diff
@@ -77,22 +77,40 @@ The primary objective of this document is to develop a flutter dating app that a
    - onNextCallBack: ''
    - onRejectCallBack:''
 
-2. ### accountSetupPage1
+2. ### enterBasicDetails
 
    <!-- #### accountSetupPage1 -->
 
    Enter User basic details such as name and date of birth
 
    -Routing
 
-   - Route:'/accountSetupPage1'
+   - Route:'/enterBasicDetails'
    - navigate to:'/accountSetupPage2'
 
    -Actions
 
    - Enter name in Textfield and date of birth
    - Storing the State of the Textfield(Persistent).
 
+3. ### selectGender
+
+   <!-- #### accountSetupPage1 -->
+
+   allow user to select gender in details.
```

Comment:
- Since we are using Navigator for routing, back click will already pop back to previous screen. 
- take onNext Callback because the same page will be used by edit profile